doc/README.md

```diff
@@ -2,27 +2,43 @@
 
 ## About Flame
 
-Flame is a modular Flutter game engine that provides a complete set of out-of-the-way solutions for games, taking advantage of the powerful infrastructure provided by Flutter but simplifying the code you need to build your projects.
+Flame is a modular Flutter game engine that provides a complete set of out-of-the-way solutions for
+games. It takes advantage of the powerful infrastructure provided by Flutter but simplifies the code
+you need to build your projects.
 
-It empowers you with a simple yet effective game loop implementation, and all the surrounding utilities you might need to use in a game: images, sprites, sprite sheets, animations, audio, input, components/object system, et cetera.
+It provides you with a simple yet effective game loop implementation, and all the surrounding
+utilities that you might need to use in a game: input, images, sprites, sprite sheets, animations,
+audio, components/object systems, collision detection, physics and so on.
 
 You can pick and choose whichever parts you want, as they are all somewhat independent and modular.
 
-We also support easy-to-use integrations with famous third parties and tools, like Riveapp, tiled, Google Play Games, gamepad support and more.
+We also support easy-to-use integrations with famous third party libraries and tools, like Riveapp,
+tiled, Google Play Games, Forge2D (Box2D for Dart), physical gamepads and more.
 
-The engine and ecosystem is being constantly improved by the community, so please feel free to reach out, open issues, PRs and make suggestions. Give us a star if you want to help give the engine exposure and grow the community.
+The engine and its ecosystem is constantly being improved by the community, so please feel free to
+reach out, open issues, PRs and make suggestions.
+Give us a star if you want to help give the engine exposure and grow the community :)
 
 ## Installation
 
 Put the pub package as your dependency by dropping the following in your `pubspec.yaml`:
```

Comment:
Imho we should just reference to the pubdev installation instructions instead of constantly having to update this in our README(which we often forget, as showcased here)

Reply:
Updated with pub.dev reference and remove version